src/DownloadCommand.php

```diff
@@ -32,29 +32,31 @@ protected function execute(InputInterface $input, OutputInterface $output)
         $this->loadConfig($input);
 
         $vcards = array();
-
-        $progress = new ProgressBar($output);
+        if ($input->getOption('image')) {
+            $substitutes[] = 'PHOTO';
+        }
+        else {
+            $substitutes = [];
+        }
 
         if ($filename = $input->getArgument('filename')) {
             // read from file
             $vcards = json_decode(file_get_contents($filename));
-
             if (!is_array($vcards)) {
                 throw new \Exception(sprintf('Could not read unparsed vcards from %s', $filename));
             }
         }
         else {
             // download
             foreach($this->config['server'] as $server) {
+                $progress = new ProgressBar($output);
                 error_log("Downloading vCard(s) from account ".$server['user']);
                 $backend = backendProvider($server);
-
```

Comment:
Du hast was gegen Leerzeilen? Ist Geschacckssache M)

Reply:
Ne, aber ich weiß immer nicht wo du welche haben möchtest. Ich erinnere mich dass ich in der Vergangenheit welche gesetzt hatte und sie dir da nicht gefielen.
Gibts da 'ne Regel oder best practice?
Habe jetzt in donload und run nach dem ErrorLog wieder ne Zeile eingefügt

Reply:
Nur gefühlt. Um Zusammengehöriges i.S. der Lesbarkeit abzugrenzen.